index.html

```diff
@@ -0,0 +1,181 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles/main.css">
+    <title>Mobile portfoilo version</title>
+</head>
+<body>
+    <div class="container">
+    <header class="portfo-header">
+         <a href="#" class="mylogo"> My Logo</a>
+        <nav class="menu-button"  onclick="toggle()">
+            <img id="hamburger-btn" src="images/ham-btn.png" alt="menu">
+        </nav>
+    </header>
+    <section class="hero">
+        
+        <div class="hero-text">
+            <h1>I’m Beyk
+                Glad to see you!</h1>
+                    <h2>    I’m a software developer! </h2> 
+                <p>
+                     I can help you build a product ,
+                     feature or website Look through some of my work and experience!
+                    If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                </p>
+                <a href="#">LET’S CONNECT</a>
+        </div>
+        <div class="hero-socialmedia-icons">
+            <ul>
+                <li><a href="#"><img src="images/twitter.png" alt="twitter"></a></li>
+                <li><a href="#"><img src="images/linkdin.png" alt="linkdin"></a></li>
+                <li><a href="#"><img src="images/medium.svg" alt="medium"></a></li>
+                <li><a href="#"><img src="images/github.svg" alt="github"></a></li>
+                <li><a href="#"><img src="images/angellist.svg" alt="angellist"></a></li>
+        </ul>
```

Comment:
- [ ] Please make sure these images are aligned correctly, as on the template.
![](https://i.imgur.com/tLDZZbc.png)

Reply:
thank you,
 I fix it